pybossa/util.py

```diff
@@ -955,18 +955,19 @@ def valid_or_no_s3_bucket(task_data):
 
 
 def can_update_user_info(current_user, user_to_update):
+    disable_fields = {'user_type': 'You must be an admin or subadmin to edit this.'}
+    hidden_fields = {'profile': 'You must be an admin or subadmin to view and edit this.'}
```

Comment:
1. are these messages used somewhere?
2. if field is hidden, will user be able to see this message?
3. is disable=False will make field editable and hidden=False make field visible? If so, should hidden_field message be only about view?

Reply:
1. Reason is not used anywhere for disable and hidden fields. Having this message is to keep consistent with what already exist in master branch https://github.com/bloomberg/pybossa/blob/c5c70354f99c14a47435a50cf372d3974fd60c7e/pybossa/forms/forms.py#L817.
2. user cannot see the message
3. we can make hidden message only about view, but if they cannot view, they cannot update either

Reply:
#2, as code is not using message, does it make sense having list instead of dict consistently for disabled and hidden fields
#3 if you think to continue keep message, should hidden field message to be updated to `You must be admin or subadmin to view this`

Reply:
Ok, I will update message to align with #3